infrastructure/main.tf

```diff
@@ -220,6 +220,8 @@ resource "aws_security_group" "workflows_api_handler_sg" {
     cidr_blocks      = ["0.0.0.0/0"]
     ipv6_cidr_blocks = ["::/0"]
   }
+
+  lifecycle { create_before_destroy = true }
```

Comment:
This lifecycle policy will enable the new security group to be created before the old one is destroyed, making the deployment time a lot quicker